File: CS/Form1.cs
using System.ComponentModel;
using DevExpress.XtraBars.ColorPick.Popup;
using DevExpress.XtraBars.Ribbon;
using DevExpress.XtraEditors;
using DevExpress.XtraEditors.Controls;

namespace DXSample {
    public partial class Form1 : RibbonForm {
        public Form1() {
            InitializeComponent();
        }
        void colorPickEditPopupControl_QueryResultValue(object sender, QueryResultValueEventArgs e) {
            object res = e.Value;
        }
        void colorPickEditPopupControl_Properties_QueryPopUp(object sender, CancelEventArgs e) {
            this.colorPickEditPopupContainerControl1.UpdateLayout();
        }
        void colorPickEditPopupContainerControl1_ResultColorChanged(object sender, ColorChangedEventArgs e) {
            colorPickEditPopupControl.EditValue = e.Color;
            colorPickEditPopupControl.ClosePopup();
        }
        void colorPickEditPopupControlContainer1_ResultColorChanged(object sender, DevExpress.XtraBars.ColorPick.Popup.ColorChangedEventArgs e) {
            string msg = string.Format("Selected Color: {0}", this.colorPickEditPopupControlContainer1.ResultColor);
            XtraMessageBox.Show(msg, "Color Info");
        }
    }
}

File: VB/Form1.vb
Imports System.ComponentModel
Imports DevExpress.XtraBars.ColorPick.Popup
Imports DevExpress.XtraBars.Ribbon
Imports DevExpress.XtraEditors
Imports DevExpress.XtraEditors.Controls

Namespace DXSample
	Partial Public Class Form1
		Inherits RibbonForm
		Public Sub New()
			InitializeComponent()
		End Sub
		Private Sub colorPickEditPopupControl_QueryResultValue(ByVal sender As Object, ByVal e As QueryResultValueEventArgs) Handles colorPickEditPopupControl.QueryResultValue
			Dim res As Object = e.Value
		End Sub
		Private Sub colorPickEditPopupControl_Properties_QueryPopUp(ByVal sender As Object, ByVal e As CancelEventArgs) Handles colorPickEditPopupControl.Properties.QueryPopUp
			Me.colorPickEditPopupContainerControl1.UpdateLayout()
		End Sub
		Private Sub colorPickEditPopupContainerControl1_ResultColorChanged(ByVal sender As Object, ByVal e As ColorChangedEventArgs) Handles colorPickEditPopupContainerControl1.ResultColorChanged
			colorPickEditPopupControl.EditValue = e.Color
			colorPickEditPopupControl.ClosePopup()
		End Sub
		Private Sub colorPickEditPopupControlContainer1_ResultColorChanged(ByVal sender As Object, ByVal e As DevExpress.XtraBars.ColorPick.Popup.ColorChangedEventArgs) Handles colorPickEditPopupControlContainer1.ResultColorChanged
			Dim msg As String = String.Format("Selected Color: {0}", Me.colorPickEditPopupControlContainer1.ResultColor)
			XtraMessageBox.Show(msg, "Color Info")
		End Sub
	End Class
End Namespace
